collector/time.go

```diff
@@ -20,14 +23,16 @@ type TimeCollector struct {
 	NTPServerOutgoingResponsesTotal  *prometheus.Desc
 }
 
-func newTimeCollector() (Collector, error) {
-	if getWindowsVersion() <= 6.1 {
-		return nil, errors.New("Windows version older than Server 2016 detected. The time collector will not run and should be disabled via CLI flags or configuration file")
+func newTimeCollector(logger log.Logger) (Collector, error) {
+	const subsystem = "time"
+	logger = log.With(logger, "collector", subsystem)
 
+	if getWindowsVersion(logger) < 100 {
```

Comment:
Looking at the documentation for `getWindowsVersion`, a value of `100` is correct here. However it might be best to move this fix into a separate commit for increased visibility if we need to revisit the value in the future.

Reply:
You are correct.